cli/src/cmd/forge/script.rs

```diff
@@ -79,6 +78,18 @@ pub struct ScriptArgs {
 
     #[clap(flatten, next_help_heading = "EVM OPTIONS")]
     pub evm_opts: EvmArgs,
+
+    #[clap(
+        long,
+        help = "resumes previous transaction batch. DOES NOT simulate execution. respects nonce constraint "
+    )]
+    pub resume: bool,
+
+    #[clap(
+        long,
+        help = "resumes previous transactions batch. DOES NOT simulate execution. DOES NOT respect nonce constraint"
```

Comment:
What's the difference between this and `--resume`? (clarify in help text)

Reply:
tried to give a better explanation, lmk

Reply:
much better